spec/rubocop/cop/rspec/verified_double_reference_spec.rb

```diff
@@ -1,10 +1,24 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::RSpec::VerifiedDoubleReference do
-  described_class::RESTRICT_ON_SEND.each do |verified_double|
-    describe verified_double do
-      let(:cop_class) { described_class }
+  verified_doubles = %i[
+    class_double
+    class_spy
+    instance_double
+    instance_spy
+    mock_model
+    object_double
+    object_spy
+    stub_model
+  ]
+
+  # described_class is overwritten in the inner describe block
+  # rubocop:disable RSpec/DescribedClass
+  let(:cop_class) { RuboCop::Cop::RSpec::VerifiedDoubleReference }
+  # rubocop:enable RSpec/DescribedClass
 
+  verified_doubles.each do |verified_double|
+    describe verified_double do
```

Comment:
If you give a string argument to `describe` – e.g. just `describe verified_double.to_s do`, then described_class does not get overwritten, and lines 15-18 can be removed.

Reply:
Fixed with
```diff
- verified_doubles = %i[
+ verified_doubles = %w[
 ```

Reply:
TIL 🙌